# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sirkull/pyzor/fixinterpreter.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN pyzor-0.4.0.orig/scripts/pyzor pyzor-0.4.0/scripts/pyzor
--- pyzor-0.4.0.orig/scripts/pyzor	2002-06-08 19:24:20.000000000 -0400
+++ pyzor-0.4.0/scripts/pyzor	2003-05-12 11:09:12.000000000 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
 
 import pyzor.client
 pyzor.client.run()
diff -urN pyzor-0.4.0.orig/scripts/pyzord pyzor-0.4.0/scripts/pyzord
--- pyzor-0.4.0.orig/scripts/pyzord	2002-09-07 23:33:44.000000000 -0400
+++ pyzor-0.4.0/scripts/pyzord	2003-05-12 11:09:17.000000000 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
 
 import os
 import os.path

